packages/core-utils/src/time.js

```diff
@@ -63,7 +104,8 @@ export function formatDurationWithSeconds(seconds) {
  * @returns {string} formatted text representation
  */
 export function formatTime(ms, options) {
-  return moment(ms + (options && options.offset ? options.offset : 0)).format(
+  return format(
+    ms + (options && options.offset ? options.offset : 0),
```

Comment:
Opportunity for optional chaining? I'm not sure if that's configured here though (i.e., needs any babel plugins).

```suggestion
    ms + (options?.offset || 0),
```

Reply:
Works perfectly! Refactored in 6db148be6701f51c7a26b67765a8b15409bb63de